build-config.h_cmake_in

```diff
@@ -0,0 +1,31 @@
+#ifndef BUILD_CONFIG_h
+#define BUILD_CONFIG_H
+
+/* Name of package */
+#define PACKAGE "gromit-mpx"
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT "Christian Beier <dontmind@freeshell.org>"
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME "gromit-mpx"
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING "gromit-mpx ${version}"
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME "gromit-mpx"
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL "https://github.com/bk138/gromit-mpx"
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "${version}"
+
+/* Define to 1 if you have the ANSI C header files. */
```

Comment:
me thinks this can be left out as it's a) not used anywhere and b) leaving this to 1 unconditionally is wrong to me anyway...